pegasus/sites.v3/code.org/views/lms/lms_editorial_cards.haml

```diff
@@ -0,0 +1,34 @@
+:ruby
+  card_list = [
+    {
+      image: "/images/lms/editorial01.png",
+      title: hoc_s("lms_page.reasoning.title_01"),
+      desc: hoc_s("lms_page.reasoning.desc_01")
+    },
+    {
+      image: "/images/lms/editorial02.png",
+      title: hoc_s("lms_page.reasoning.title_02"),
+      desc: hoc_s("lms_page.reasoning.desc_02")
+    },
+    {
+      image: "/images/lms/editorial03.png",
+      title: hoc_s("lms_page.reasoning.title_03"),
+      desc: hoc_s("lms_page.reasoning.desc_03")
+    },
+    {
+      image: "/images/lms/editorial04.png",
+      title: hoc_s("lms_page.reasoning.title_04"),
+      desc: hoc_s("lms_page.reasoning.desc_04")
+    },
+  ]
+
+.editorial-cards-wrapper-2x2
+  - card_list.each do |card_list|
+    .flex-container
+      %figure
+        %img{src: "#{card_list[:image]}", alt: ""}
```

Comment:
Does this need to be interpolated?

Reply:
(same question in the other views)

Reply:
These need to stay as-is. I tried refactoring the quotes and the page no longer loaded. Fwiw, I copied this format from https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/views/administrators/admins_editorial_cards.haml

Reply:
There's an example [here](https://github.com/code-dot-org/code-dot-org/blob/478de6f89be9d4c3a3c55a0b8fd1e128145d3a3e/pegasus/sites.v3/code.org/views/coding_with_ai/coding_with_ai_editorial_cards.haml#L24) w/o the image interpolated — I vaguely remember this happening somewhere else too. If this does indeed work w/o interpolation I can update the Pegasus Docs.

Reply:
I was able to update these to remove the quotes, and went back to remove them from the places I copied from as well (in the administrators views folder). When I went back to look for more cases (via searching [:img]}" and [:image]}" I found about 25 examples in other haml files, so am going to make a ticket to go back and update those later instead of adding them here.

Reply:
(Jira is currently down but I'll link that ticket once I can make it)

Reply:
Here's that ticket! https://codedotorg.atlassian.net/browse/ACQ-1951

Reply:
haha sorry, didn't mean to send you down such a rabbit hole!

Reply:
It's all good! With each new page we make Pegasus a *little better